main.go

```diff
@@ -412,10 +420,13 @@ func doMain(args []string) error {
 	if err != nil {
 		return err
 	}
-	// Do not run anything else if update-check is triggered.
+	// Do not run anything else if update-check or inventory-update is triggered.
 	if *runOptions.updateCheck {
 		return updateCheck(exec.Command("kill", "-USR1"), exec.Command("systemctl", "show", "-p", "MainPID", "mender"))
 	}
+	if *runOptions.updateInventory {
+		return updateCheck(exec.Command("kill", "-USR2"), exec.Command("systemctl", "show", "-p", "MainPID", "mender"))
+	}
```

Comment:
Is it a valid behavior to specify both `check-update` and `send-inventory`?

Reply:
Good eyes :eyes:

Reply:
And that is a no.

Reply:
Fixed in https://github.com/mendersoftware/mender/pull/336/commits/fabeaad473da28d76c172493a10b3ef198196b62